sqlflow.proto

```diff
@@ -1,49 +1,51 @@
 syntax = "proto3";
 
+package main;
+
 service SQLFlow {
-  rpc Run(RunRequest) returns (RunReply) {}
+  // Run runs a SQL statement and returns a stream of RunReply.  Some
+  // SQL statements like `USE database` returns only a success status.
+  // In such cases, the return stream contains only one RunReply with
+  // RunReply.error set.  Some other SQL statements like `SELECT ...`
+  // returns a table in addition to the status, and the table might be
+  // big.  In such cases, Run returns a stream of RunReply -- the
+  // first one has RunReply.schema set, and all RunReplies have
+  // RunReply.error and RunReply.columns set -- each RunReply doesn't
+  // exceed the maximum message size
+  // (https://stackoverflow.com/a/34186672/724872).
+  rpc Run(RunRequest) returns (stream RunReply) {}
```

Comment:
```suggestion
  rpc Run(RunRequest) returns (stream RunResponse);
```
Suggest to follow request/response naming convention.
Remove empty {} and add ;